src/app/component/auth/new-sign-up/new-sign-up.component.html

```diff
@@ -188,7 +188,7 @@ <h2 class="subtitle-text">
                  class="sign-up-google-img"
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/82157338-8b1b8700-9889-11ea-94d2-c2d6252e9975.png)
You should add some fixes to the 'Forgot password' modal

Reply:
Please, revert changes in the 'sign up' form controller

Reply:
fixed